### PR TITLE
NO_PROXY local registry

### PR DESCRIPTION
**Pull request by areed**

### CODE REVIEW

File: install_scripts/templates/common/ip-address.sh [hunk 1]
```diff
@@ -178,3 +179,16 @@ isValidIpv4() {
...
+# Returns the ip portion of an address.
+# Globals:
+#   None
+# Arguments:
+#   ADDRESS
+# Returns:
+#   PARSED_IPV4
+#######################################
+parseIpv4FromAddress() {
+    PARSED_IPV4=$(echo "$1" | grep --only-matching '[0-9][0-9]*\.[0-9][0-9]*\.[0-9][0-9]*\.[0-9][0-9]*')
```
Comment (emosbaugh) [inline install_scripts/templates/common/ip-address.sh]: this is good but we do support ipv6 i think or at least we did in the past. maybe we wanna merge and add a ticket for support?

File: install_scripts/templates/common/ip-address.sh [hunk 2]
```diff
@@ -178,3 +179,16 @@ isValidIpv4() {
...
+
+#######################################
+# Returns the ip portion of an address.
+# Globals:
+#   None
+# Arguments:
+#   ADDRESS
+# Returns:
+#   PARSED_IPV4
+#######################################
```
Comment (emosbaugh) [inline install_scripts/templates/common/ip-address.sh]: i would prefer globals right above function here
example https://github.com/replicatedcom/install-scripts/blob/master/install_scripts/templates/common/common.sh#L49

File: install_scripts/templates/common/ip-address.sh (hunk 1 above)
Comment (areed) [reply install_scripts/templates/common/ip-address.sh]: OK

File: install_scripts/templates/common/ip-address.sh (hunk 2 above)
Comment (areed) [reply install_scripts/templates/common/ip-address.sh]: moved

**Pull request merged**